data/os/RedHat.yaml

```diff
@@ -0,0 +1,6 @@
+---
+sensu::plugins::dependencies:
```

Comment:
the dependencies are missing from the spec tests

Reply:
Added these to unit tests.